src/c++/perf_analyzer/client_backend/openai/openai_client.cc

```diff
@@ -63,6 +63,14 @@ namespace openai {
 void
 ChatCompletionRequest::SendResponse(bool is_final, bool is_null)
 {
+  // if final response has already been sent
```

Comment:
It's more that when we measure OpenAI protocols we consider 'done' to be when `[DONE]`
message is received (as opposed to when the http message is completed by the server).

Since `generate` does not have a `[DONE]` message - we treat either `[DONE]` as complete OR the end of the message - this check is to ignore in case we have already sent it